### ISSUE TITLE
Transliteration to Cyrillic does not work

### ISSUE THREAD

**Issue by volanar** [NodeBB/NodeBB#9336]
Nodebb all versions.
Transliteration of Cyrillic characters to Latin characters does not work on all versions of nodebb. This is bad for SEO.
Example:
https://domorost.ru/category/10/%D0%B2%D0%B0%D1%88%D0%B8-%D0%B2%D0%BE%D0%BF%D1%80%D0%BE%D1%81%D1%8B

**Comment by pitaj** [comment]
Please follow the issue template.

What are you expecting to see? Why is it bad for SSO, and according to who?

**Comment by julianlam** [comment]
I'm guessing it is a browser issue, in that it is showing the escaped characters.

@volanar hover over a topic url, does it show the proper Cyrillic?

**Comment by nhlpl** [comment]
You can find some talk on this topic on both GitHub and NodeBB Community Forums.
https://github.com/NodeBB/NodeBB/issues/3097
https://github.com/NodeBB/NodeBB/issues/3129

**Comment by pitaj** [comment]
Yeah there's talk about transliteration. You seem to want it, but others prefer it this way. You haven't explained why transliteration is beneficial to you besides vague "SEO reasons". Nor have you given an example of what you expect to see vs what already exists.

**Comment by nhlpl** [comment]
It looks like it can be done via a plug-in and nodebb-plugin-ns-slugify seems to be a good example.

**Comment by volanar** [comment]
> Ага, есть разговоры о транслитерации. Кажется, вы этого хотите, но другие предпочитают такой способ. Вы не объяснили, почему транслитерация вам выгодна, помимо расплывчатых «причин SEO». Вы также не привели пример того, что вы ожидаете увидеть, и того, что уже существует.

Everyone loves beautiful URLs, including search engines. This is one of the ranking factors. Cyrillic characters are not the best option for using them in URLs. Discourse and XenForo have built-in transliteration, wordpress and IPB have additional plugins. It is much better when transliteration is in the core. These are good SEO practices

**Comment by mySebbe** [comment]
Current master deliberately preserves Unicode slugs, and the test suite explicitly asserts Cyrillic, CJK, and Arabic behavior. Therefore the reported result is not a broken transliterator in core; transliteration would be an alternative URL policy with collision and redirect consequences. I suggest closing this as not-a-core-bug. If configurable transliteration is still wanted, a new issue should specify a supported server-side slug customization point plus collision, rename, and legacy-redirect behavior rather than changing Unicode defaults for every installation.
